presto-docs/src/main/sphinx/release/release-0.280.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.280
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a bug where if a query was submitted before the resource group configuration had been loaded, all queries to the cluster would fail with a ``NullPointerException``.
+* Fix bug in ``GatherAndMergeWindows`` optimization rule.
```

Comment:
What bug? May be add some detail?

Reply:
I have tried to add more details, pls check.